src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx

```diff
@@ -100,6 +100,9 @@ const WorkspacePipelineRunPage: NextPageWithLayout = (props: Props) => {
     if (isConnectionParameter(entry.type) && entry.value) {
       return entry.value;
     }
+    if (entry.type === "dataset") {
+      return entry.value;
```

Comment:
It makes not a lot of sense for a user. It's better to display a link to the dataset with its name and not its slug.